### PR TITLE
check stream specific hwaccel_args for gpu stats

### PR DESCRIPTION
**Pull request by Rob-Powell**
Previously only cameras were checked and parsed for the existance of hwaccel_args but it can also be stream specific only so i've added a check for that as well.

### CODE REVIEW

File: frigate/stats.py [hunk 1]
```diff
@@ -121,6 +121,15 @@ async def set_gpu_stats(config: FrigateConfig, all_stats: dict[str, Any]) -> Non
...
             hwaccel_args.append(args)
+     
+    for stream_input in camera.ffmpeg.inputs:
+        args = stream_input.hwaccel_args
+
+        if isinstance(args, list):
+            args = " ".join(args)
+
+        if args and args not in hwaccel_args:
+            hwaccel_args.append(args)
```
Comment (NickM-27) [inline frigate/stats.py:132]:
```suggestion
     
        for stream_input in camera.ffmpeg.inputs:
            args = stream_input.hwaccel_args

            if isinstance(args, list):
                args = " ".join(args)

            if args and args not in hwaccel_args:
                hwaccel_args.append(args)
```

Comment (NickM-27) [reply frigate/stats.py:132]: If it's at the same level as the first loop it's only going to check the last camera where you want it to check all cameras.

**Review by NickM-27 — CHANGES_REQUESTED** [review]

File: frigate/stats.py (hunk 1 above)
Comment (Rob-Powell) [reply frigate/stats.py:132]: haha thanks copy paste fail

**Review by NickM-27 — APPROVED** [review]

**Comment by blakeblackshear** [comment]
Linter still failed here

**Comment by Rob-Powell** [comment]
I ran the linter and it says its fine:
```
vscode ➜ /workspace/frigate (dev ✗) $ python3 -m black frigate/stats.py
All done! ✨ 🍰 ✨
1 file left unchanged.
```

**Review by blakeblackshear — APPROVED** [review]

**Pull request merged**